### DISCUSSION TITLE
CheckToken fail if i have the language in my Url

### DISCUSSION THREAD

**Discussion by HastaCs** [phalcon/cphalcon#15305] [Q&A]
Hi, i can acces to my web with his 2 urls:
   myweb.com/controller or myweb.com/es/controller     *myweb/{language}/controller
   
  All works fine but not the checkToken, only fails when i write the {language} in the url... when i do the CheckToken from any */{language}/controller always return false.
  
  This is in the form
`   <input type='hidden' id="token" name='<?php echo $this->security->getTokenKey() ?>' value='<?php echo $this->security->getToken() ?>' />`
this is in the javascript.
```
 const token = $('#token');
            const data = { key: token[0].name, value: token.val() }
        $.ajax({
            url: '/partnercheckin/insertaReserva',
            data: data,
            method: 'post',
            dataType: 'json',
            success: function(response) {
               
                console.log(response)
            },
            error: function(error) {
                console.log(error)
            }
        });
```

and this is in the controller
```
 $response = [
            "key" => $this->request->getPost('key'),
            "value" => $this->request->getPost('value'),
            "valido" => $this->security->checkToken($this->request->getPost('key'), $this->request->getPost('value'))
        ];
```
   
GREETINGS

**Comment by quasipickle** [comment]
What is in the generated markup?  What data is actually getting POSTed?
